PcsConfig/SeedDataTest.cs

```diff
@@ -19,6 +19,8 @@ public class SeedDataTest
         public SeedDataTest()
         {
             this.httpClient = new HttpClient();
+            // wait for config to run seed data
+            System.Threading.Thread.Sleep(60000);
```

Comment:
this doesn't seem very deterministic, why 60 seconds and not 30 or 90? tests should rely on predictable behavior.

Reply:
done